src/components/home/get-invloved/Circle.jsx

```diff
@@ -1,7 +1,21 @@
 import React from "react";
+import Image from "next/image";
+import mandala from "@/public/svg/mandala.svg";
 
-const Circle = () => {
-  return <div>Circle</div>;
+const Circle = ({ reactIcon, text }) => {
+  return (
+    <div className="flex flex-col items-center">
+      <div className="relative">
+        <Image src={mandala} alt="Mandala" className="w-full h-full" />
+        {reactIcon && (
```

Comment:
Good job demonstrating that you know how to conditionally render something in react. But that's not necessary for this issue.